packages/codemods/src/transforms/v2/web-react/__testfixtures__/modal-custom-height.output.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// @ts-ignore: No declaration -- The library is not installed; we don't need to install it for fixtures.
+import { ModalDialog } from '@lmc-eu/spirit-web-react';
+
+export const MyComponent = () => (
+  <>
+    <ModalDialog height={{
+      mobile: '400px'
+    }} />
```

Comment:
So the codemod always produces an object? Because this syntax is also supported:

```suggestion
    <ModalDialog height="400px" />
```

Reply:
It always produces object. It was easier for me to do and it is not wrong.